stories/theme.GHG.introduction_greenhouse_gases/carousel_content.json

```diff
@@ -0,0 +1,43 @@
+    },{
+        "src":"https://www.youtube.com/embed/P_OmxbB-iHg",
+        "title":"Trends in atmospheric Nitrous Oxide (N\u2082O)",
+        "caption":"\nVisualizations by: Helen-Nicole Kostis\nFor more information or to download this public domain video, go to https://svs.gsfc.nasa.gov/5117"
+    },{
+        "src":"https://www.youtube.com/embed/9V3Hhax5C1M",
+        "title":"Trends in atmospheric Methane (CH\u2084)",
+        "caption":"\nVisualizations by: Helen-Nicole Kostis, Scientific consulting by: Lesley Ott, Brad Weir\nFor more information or to download this public domain video, go to https://svs.gsfc.nasa.gov/5118"
+    },{
+        "src":"https://www.youtube.com/embed/bvK-SzcyuKs",
+        "title":"OCO-2 and Keeling Curve: Trends in global atmospheric Carbon Dioxide (CO\u2082)",
+        "caption":"\nVisualizations by: Helen-Nicole Kostis\nFor more information or to download this public domain video, go to https://svs.gsfc.nasa.gov/5119"
+    },{
+        "src":"https://www.youtube.com/embed/2xxjiu0C_-Q",
+        "title":"ODIAC: a map of human made carbon dioxide emissions",
+        "caption":"The Open-source Data Inventory for Anthropogenic CO2 (A.K.A. ODIAC) is a global high-resolution (1x1km) emission data product for fossil fuel carbon dioxide (CO2) emission. ODIAC was originally designed and developed under the Greenhouse gas Observing SATellite (GOSAT) project at Japan\u2019s National Institute for Environmental Studies (NIES). Since then, ODIAC has been maintained at the Universities Space Research Association (USRA) in collaboration with NASA, NIES and the Appalachian State University. ODIAC pioneered the combined use of space-based nighttime light data and individual power plant emission/location profiles to estimate the global spatial extent of fossil fuel CO2 emissions. ODIAC has been a key input data for NASA\u2019s carbon modeling. ODIAC has been also widely used in the international science research community for a variety of applications across key policy relevant scales (global to local).\nVisualizations by: Mark SubbaRao, Scientific consulting by: Tomohiro Oda\nFor more information or to download this public domain video, go to https://svs.gsfc.nasa.gov/5121"
```

Comment:
```suggestion
        "caption":"The Open-source Data Inventory for Anthropogenic CO₂ (A.K.A. ODIAC) is a global high-resolution (1x1km) emission data product for fossil fuel carbon dioxide (CO₂) emission. ODIAC was originally designed and developed under the Greenhouse gas Observing SATellite (GOSAT) project at Japan\u2019s National Institute for Environmental Studies (NIES). Since then, ODIAC has been maintained at the Universities Space Research Association (USRA) in collaboration with NASA, NIES and the Appalachian State University. ODIAC pioneered the combined use of space-based nighttime light data and individual power plant emission/location profiles to estimate the global spatial extent of fossil fuel CO₂ emissions. ODIAC has been a key input data for NASA\u2019s carbon modeling. ODIAC has been also widely used in the international science research community for a variety of applications across key policy relevant scales (global to local).\nVisualizations by: Mark SubbaRao, Scientific consulting by: Tomohiro Oda\nFor more information or to download this public domain video, go to https://svs.gsfc.nasa.gov/5121"
```

Reply:
Subscripted 2